packages/jellyfish-api-core/__tests__/category/loan/closeVault.test.ts

```diff
@@ -105,15 +150,10 @@ describe('Loan', () => {
     })
     await tGroup.get(0).generate(1)
 
-    // loan token setup
-    await tGroup.get(0).rpc.loan.setLoanToken({
-      symbol: 'TSLA',
-      fixedIntervalPriceId: 'TSLA/USD'
-    })
-    await tGroup.get(0).generate(1)
-
     // Mint TSLA
-    await tGroup.get(0).token.mint({ symbol: 'TSLA', amount: 30000 })
```

Comment:
actually just need to change the "mint" to "takeLoan"..
the purpose behind is we were minting the loan tokens previously, its not allowed in reality..
so instead of getting the loan token from "mint", it should be getting by "takeLoan" only

Reply:
i guess in this particular case i can do that. but some tests do have hardwired expects for amounts. i guess i will handle those seperately

Reply:
the trick is just like this
1. deposit a huge DFI amount into a "master" vault
2. then take the same loan amount to match the mint amount previously will do

Reply:
yes, basically minting loan tokens should not even be in the comments.